geniusyield-server-lib/src/GeniusYield/Server/Dex/PartialOrder.hs

```diff
@@ -376,8 +376,7 @@ type OrdersAPI =
       :> Description "Get details of an order using it's unique NFT token. Note that each order is identified uniquely by an associated NFT token which can then later be used to retrieve it's details across partial fills."
       :> "details"
       :> Capture "nft-token" GYAssetClass
-      :> Throws PodOrderNotFound
-      :> Get '[JSON] OrderInfoDetailed
+      :> UVerb 'GET '[JSON] '[WithStatus 200 OrderInfoDetailed, WithStatus 404 PodOrderNotFound]
```

Comment:
Have made a move from [`servant-checked-exceptions`](https://github.com/cdepillabout/servant-checked-exceptions#readme) to the approach mentioned [here](https://docs.servant.dev/en/stable/cookbook/uverb/UVerb.html).